content/module-reference/utility-modules/shared/filterings.md

```diff
@@ -0,0 +1,106 @@
+---
+title: filtering
+id: filtering
+applicable-version: 3.8
+tags:
+view: lighttable, darkroom, tethering, map
+---
+
+Filter the _collection_ of images shown in the lighttable view and filmstrip panel using image attributes.
+
+Once you have set the _collection_ of images with the [collections](./collections.md) module, you can refilter and sort it by setting some filters based on image attributes.
+
+The filters can optionally be "pinned" into the top toolbar for a quick access.
+
+# filtering attributes
+
+The images attributes used to setup filters are defined in [collections module](./collections.md/#filtering-attributes).
+
+Note that currently only a subset of this filters are implemented. More is expected in next versions.
+
+# filters controls
+
+## combining multiple filters
+
+When you use multiple filters, the first item on the filter header allow you to define how the filter is combined with the previous one.
+
+and
+: narrow down search. An image is only retained if it also fulfills the new criteria.
+
+or
+: add more images. Images from the _collection_ that fulfill the new criteria are added.
+
+and not
+: exclude images. Images that fulfill the new criteria are removed.
+
+## removing or deactivating filters
+
+You can remove completely or deactivate temporarily a specific filter using the buttons on the right of the header.
+
+## pinning into the top toolbar
+
+The pin togglebutton on the right of the header allow you to pin a _clone_ of the filter into the top toolbar. The filter and its clone stay synchronized all the time. To avoid unwanted actions, pinned filters can't be removed or deactivated. If you want to do so, un-pin the filter first.
+
+# filters specific widgets
+
+## color labels
+
+You can filter images by their colorlabels by using the dedicated filter which expose a widget where you can click on the colorlabel you want to search for.
```

Comment:
My proposal on this.

The colorlabels widget exposes the colors you may want to filter on. 
The simple way to use it is to click on the colors the filtered images must have. The selected colors buttons appear full colored. Example: Red AND Yellow.
If you want to get the images which have one color or another, click on the rightest button to toggle the combination from AND (`∩`) to OR (`∪`). Example: Blue OR Green.
You may also want to exclude a color. To do this, Ctrl-Click on the corresponding color which appears with cross on it. Example: Red AND NOT green, or, NOT Blue OR Yellow.
Last, if you want to include or exclude all the colors at once, Click or Ctrl-Click on the _grey_ button.

I think the description from simple to complex may help the reader to get it right. But I let @elstoc make the final writing. :wink: